app/dojo.py

```diff
@@ -316,16 +314,14 @@ def reallocate_person(self, emailaddress, new_roomname):
 							current_room = this_office_space
 							if current_room == new_room:
 								return (text_format.CRED + "\nCannot reallocate\
-								 to the same OFFICE\n" 
-									+ text_format.CEND)
+								 to the same OFFICE\n" + text_format.CEND)
 							else:
 								current_room.occupants.remove(person_reallocating)
 								new_room.occupants.append(person_reallocating)
-								return (text_format.CBOLD + "\n{}-{} was succesfully\
-								 reallocated to {} {}\n"
-									.format(person_reallocating.role, person_reallocating.name,
-									 new_room.room_type,new_room.name) 
-									+ text_format.CEND)
+								return (text_format.CBOLD + "\n{}-{} was succesfully \
+								 reallocated to {} {}\n".format(person_reallocating.role,
+								  person_reallocating.name, new_room.room_type,new_room.name) 
+								 + text_format.CEND)
 			else:
 				return (text_format.CRED + "\nThe room {} is full! Cannot reallocate {}-{}!\n"
 					.format(new_roomname, person_reallocating.role, person_reallocating.name)
```

Comment:
I don't see where you have are comparing the previous room type to the new one, it should not be possible to reallocate from an office to a living. The rooms, one reallocating from and to, should be of same type.